cmd/exchange_template/exchange_template.go

```diff
@@ -28,7 +29,6 @@ const (
 type exchange struct {
 	Name        string
 	CapitalName string
-	Variable    string
 	REST        bool
 	WS          bool
 	FIX         bool
```

Comment:
![image](https://github.com/user-attachments/assets/83896cea-e5fc-4c15-9b77-cda49f8342cf)

Can purge FIX as it does nothing, until we actually start implementing that func with its own streaming service then it can be re added later.

Suggestion/optional: We can purge rest as all exchanges I have come across do have rest support as default.

Reply:
Thank you for the suggestion. And, I think we should keep `--rest` for now but I have removed the fix option